client/account-recovery/lost-password/lost-password-form/test/index.jsx

```diff
@@ -24,22 +23,22 @@ describe( 'LostPassword', () => {
 		useFakeDom();
 
 		it( 'submit button shuold be disabled if user login is blank', function() {
-			const wrapper = mount( <LostPasswordFormComponent className="test__test" /> );
+			const wrapper = mount(
+				<LostPasswordFormComponent className="test__test"
+					userLogin={ '' }
+				/> );
 
 			wrapper.find( '.lost-password-form__user-login-input' ).node.value = '';
-			wrapper.find( '.lost-password-form__user-login-input' ).simulate( 'change' );
 			expect( wrapper.find( '.lost-password-form__user-login-input' ).prop( 'disabled' ) ).to.not.be.ok;
 			expect( wrapper.find( '.lost-password-form__submit-button' ).prop( 'disabled' ) ).to.be.ok;
 		} );
 
-		it( 'should be disabled when submit button clicked', function() {
-			const wrapper = mount( <LostPasswordFormComponent className="test__test" /> );
+		it( 'should be disabled when isRequesting is on', function() {
```

Comment:
True. That's more explicit. Fixed by a part of cf2902b967bd283e54354399a1222d1e80f6a7c8.